### DISCUSSION TITLE
Sell/Grab & Home are bugged?

### DISCUSSION THREAD

**Discussion by Dexteriority** [winstxnhdw/lc-hax#299] [Q&A] [answered]
So for me whenever i used a Sell command, my speed are decreased to how many value i type, for example 1K, my weight are sooo heavy that i started to walk backwards automatically. Whenever i used Grab command, my speed increased to how many value items grabbed from Dungeon, my weight so light that i can speed walk but only backwards, im removing No Weight mod since to make it looks legit XD. So how to fix it?

and if i use /Home with not players name instead i use "1" "2", some npc spawned with Player costume and not moving at all, what actually happened there?

**Comment by winstxnhdw** [comment, marked as answer]
You can fix it by adding NoWeightMod back. But jokes aside, I can probably do something about this.

> and if i use /Home with not players name instead i use "1" "2", some npc spawned with Player costume and not moving at all, what actually happened there?

The game naturally adds player dummies if there’s no player controlling the model, so you are just teleporting an empty shell back. `/home` doesn’t do anything special. It’s using the exact same code that’s used to teleport a person home if you were to do it yourself manually. So if it was happening in your game, it would’ve happened with the teleporter as well.

**Reply by Dexteriority** [reply]
Got it. Thanks

**Reply by winstxnhdw** [reply]
This is now fixed in latest.

**Reply by Dexteriority** [reply]
Yup! code now works as person! Great work!!!
